pkgs/development/tools/go-minimock/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "go-minimock";
+  version = "3.0.8";
+
+  src = fetchFromGitHub {
+    owner = "gojuno";
+    repo = "minimock";
+    rev = "v${version}";
+    sha256 = "0r0krbwvx5w1z0yv2qqi92irbsfhkvwvaigy350cvcz9gmcppj4h";
+  };
+
+  vendorSha256 = "1macwm6hybjinwnx62v146yxydcn5k5r587nxwkf4ffy76s2m3jc";
+
+  doCheck = false;
```

Comment:
A comment as to why tests are disabled would be nice.

Reply:
Currently it amounts to the same thing since there are no tests in cmd/minimock. Let me see if I can get it to run the tests from the root directory.

Reply:
Updated, looks like it's getting all the tests now. Thank you!